### ISSUE TITLE
When numprocesses = 0 initially, I cannot add spawn new processes through the web console

### ISSUE THREAD

**Issue by radeinla** [circus-tent/circus#959]
I am unsure if this is also true for the command line or if this is considered a bad practice. My goal for setting the `numprocesses = 0` initially is that I have a celery worker/beat that I want to only start on demand during every deploy and start as I wish (after checking everything else). Not sure if you guys can point me to a better way to do this.

**Comment by k4nar** [comment]
I think you should use the `autostart` option instead. Look here: http://circus.readthedocs.org/en/latest/for-ops/configuration/#watcher-name-as-many-sections-as-you-want

**Comment by radeinla** [comment]
Ah! Didn't notice that option. Not sure if its a bug (what I encountered though). Seems like I should still be able to set the numprocesses to 0 and modify at runtime. If its done for some reason, would be nice for circus to give me a warning when initially setting it to 0.

**Comment by k4nar** [comment]
Yeah, it sounds like a bug. I don't have much time to work on it, but I could take a look if I had a simple test case :) .

**Comment by k4nar** [comment]
If anyone want to work on this, please try to provide a failing test case first.
